frontend/pages/entities/[slug].tsx

```diff
@@ -14,11 +28,37 @@ interface EntityProps {
  * A page that renders the HTML of a single entity.
  */
 const EntityDetailPage: FC<EntityProps> = ({ entity }: EntityProps) => {
+  const hasRelatedQuotes = (entity.relatedQuotes && entity.relatedQuotes.length) || undefined;
+  //const hasRelatedEntities = entity.relatedEntities && entity.relatedEntities.length || undefined; #TODO: fix in entity schema
+
   return (
     <Layout title={entity.name}>
       <ModuleContainer>
         <ModuleDetail module={entity} />
       </ModuleContainer>
+      <ModuleContainer>
+        {hasRelatedQuotes && (
+          <CardHeader
+            title={`Quotes from ${entity.name}:`}
+            style={{ textAlign: "center" }}
+          ></CardHeader>
+        )}
+        {hasRelatedQuotes &&
+          entity.relatedQuotes?.map((relatedQuote) => (
+            <Link href={`/quotes/${relatedQuote.slug}`} key={relatedQuote.slug} passHref>
+              <a>
+                <EntityRelatedQuoteCard raised>
+                  {relatedQuote.dateString && (
+                    <CardHeader title={relatedQuote.dateString}></CardHeader>
+                  )}
+                  <CardContent>
+                    <HTMLEllipsis unsafeHTML={relatedQuote.bite} maxLine="3" basedOn="words" />
+                  </CardContent>
+                </EntityRelatedQuoteCard>
+              </a>
+            </Link>
+          ))}
+      </ModuleContainer>
     </Layout>
```

Comment:
This can be simplified a little. Currently, we (1) check `hasRelatedQuotes` twice and (2) render an empty `ModuleContainer` even if there are no quotes. I think we should use this structure instead:
```jsx
<Layout>
  <>
    <ModuleContainer/>  // first one
    {entity.relatedQuotes?.length > 0 && (
      <ModuleContainer>
        <CardHeader/>
        {entity.relatedQuotes.map(...)}
      </ModuleContainer>
    }
  </>
</Layout>
```
I'm not 100% sure, but I also think that `ModuleContainer` only provides some margins, so it's possible you can just place the new block right after `ModuleDetail` instead of creating a second `ModuleContainer`.